presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -207,15 +207,16 @@ void checkSplitsForBatchTask(
 }
 
 struct ZombieTaskStats {
-  const std::string taskId;
-  const std::string taskInfo;
-
-  explicit ZombieTaskStats(const std::shared_ptr<exec::Task>& task)
-      : taskId(task->taskId()), taskInfo(task->toString()) {}
-
-  std::string toString() const {
-    return SystemConfig::instance()->logZombieTaskInfo() ? taskInfo : taskId;
-  }
+  const std::string info;
+  const long numExtraReferences;
+
+  ZombieTaskStats(
+      const std::shared_ptr<exec::Task>& task,
+      long _numExtraReferences)
```

Comment:
```
s/long/uint32_t/ why long? Thanks!
```

Reply:
@xiaoxmeng
Simply based on where we get them from:
`const auto prestoTaskRefCount = prestoTask.use_count();`
which comes from
`long use_count() const _NOEXCEPT`